listview: don't autoscroll on column resize (fixes #527)
https://github.com/cboxdoerfer/fsearch/commit/3b816724f345b9c168644b8fedbc9fc42bb82824

Changing the column width while the list is scrolled to the bottom,
triggers autoscroll (i.e. the list slowly moves to the top).
This happened because the listview was behaving like in rubberband mode.

--- a/src/fsearch_list_view.c
+++ b/src/fsearch_list_view.c
@@ -1083,6 +1083,9 @@
     //     return;
     // }
 
+    if (view->bin_drag_mode == FALSE) {
+        return;
+    }
     update_rubberband_selection(view);
     add_vertical_autoscroll_timeout(view);
     gtk_widget_queue_draw(GTK_WIDGET(view));
